pci/src/vfio.rs

```diff
@@ -1497,10 +1580,22 @@ impl BusDevice for VfioPciDevice {
     }
 }
 
+// TODO(sdake) examine prefetch bit as described on page 704 implementation notes.
```

Comment:
still need to look into the prefetch bit settings. The host has differing values from the guest.